todos-mongokitten-openapi/Sources/TodosOpenAPI/openapi.yaml

```diff
@@ -0,0 +1,128 @@
+openapi: '3.1.0'
```

Comment:
Is it considered good practice to separate the OpenAPI generated code into another module?

Reply:
I've found it to be good practice, as Swift tends to generate errors for the types in that file if it's not in a dependency. It gets confused about the file not existing